instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -2,15 +2,76 @@ import React from "react";
 import Comments from "./Comments";
 import CommentInput from "./CommentInput";
 
-const CommentSection = props => {
-  return (
-    <div>
-      {props.comments.map(comment => (
-        <Comments key={comment.id} comments={comment} />
-      ))}
-      <CommentInput />
-    </div>
-  );
-};
+//styles
+import "./CommentSection.css";
+
+class CommentSection extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentsData: [],
+      text: ""
+    };
```

Comment:
```suggestion
likes: 0
    };
```
Initialize likes here anyway, even though it's being set in CDM,  for readability